File: cepf-api/src/main/java/br/com/rmc/cepf/pessoa/PessoaService.java
package br.com.rmc.cepf.pessoa;

import org.springframework.stereotype.Service;

import br.com.rmc.BaseService;

@Service
class PessoaService extends BaseService<Pessoa> {
	
	private static final long serialVersionUID = 4044055203125228964L;
	
	public PessoaService(PessoaRepository repository) {
		super(repository);
	}
}

File: cepf-ui/src/app/pessoa/pessoa.service.ts
import {Injectable} from '@angular/core';
import {HttpClient} from '@angular/common/http';

import {BaseService} from '../../angular-framework/service/base-service';
import {Pessoa} from './pessoa';

@Injectable({
  providedIn: 'root'
})
export class PessoaService extends BaseService<Pessoa> {

  constructor(httpClient: HttpClient) {
    super('/pessoas', httpClient);
  }
}
